client/pages/product/addproduct.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import { axiosInstance } from '../../configs/axios';
+import { Router } from 'next/router';
+
+const CreateProduct = () => {
+  const [product, setProduct] = useState({
+    name: '',
+    description: '',
+    image: '',
+    price: '',
+    intervals: 'monthly', // Default interval
+  });
+
+  /* Use state to store the token
```

Comment:
What is this? You've just commented the code, but where's the fix? Plus do not leave the commented code, remove it.